klayout/drc/rule_decks/dualgate.drc

```diff
@@ -0,0 +1,87 @@
+################################################################################################
+# Copyright 2022 GlobalFoundries PDK Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################################
+
+if FEOL
+    #================================================
+    #--------------------DUALGATE--------------------
+    #================================================
+
+    # Rule DV.1: Min. Dualgate enclose DNWELL. is 0.5µm
+    logger.info("Executing rule DV.1")
+    dv1_l1 = dualgate.enclosing(dnwell, 0.5.um, euclidian).polygons(0.001)
+    dv1_l2 = dnwell.not_outside(dualgate).not(dualgate)
+    dv1_l  = dv1_l1.or(dv1_l2)
+    dv1_l.output("DV.1", "DV.1 : Min. Dualgate enclose DNWELL. : 0.5µm")
+    dv1_l1.forget
+    dv1_l2.forget
+    dv1_l.forget
+
+    # Rule DV.2: Min. Dualgate Space. Merge if Space is less than this design rule. is 0.44µm
+    logger.info("Executing rule DV.2")
+    dv2_l1  = dualgate.space(0.44.um, euclidian).polygons(0.001)
+    dv2_l1.output("DV.2", "DV.2 : Min. Dualgate Space. Merge if Space is less than this design rule. : 0.44µm")
+    dv2_l1.forget
+
+    # Rule DV.3: Min. Dualgate to COMP space [unrelated]. is 0.24µm
+    logger.info("Executing rule DV.3")
+    dv3_l1  = dualgate.separation(comp.outside(dualgate), 0.24.um, euclidian).polygons(0.001)
+    dv3_l1.output("DV.3", "DV.3 : Min. Dualgate to COMP space [unrelated]. : 0.24µm")
+    dv3_l1.forget
+
+    # rule DV.4 is not a DRC check
```

Comment:
what is DV.4 then?

Reply:
https://gf180mcu-pdk.readthedocs.io/en/latest/physical_verification/design_manual/drm_07_07.html

Reply:
maybe add the url in the comment?

Reply:
Done